quinn-proto/src/endpoint.rs

```diff
@@ -844,7 +851,7 @@ pub enum ConnectError {
 }
 
 #[derive(Default, Debug)]
-struct ResetTokenTable(HashMap<SocketAddr, HashMap<ResetToken, ConnectionHandle>>);
+struct ResetTokenTable(FxHashMap<SocketAddr, FxHashMap<ResetToken, ConnectionHandle>>);
```

Comment:
Using a `FxHashMap` here but a regular `HashMap` for `connection_remotes` seems inconsistent. I'm uncertain whether there's a meaningful collision attack risk with socket addrs, since they're only partially controlled, but we should be consistent either way.

Reply:
Agreed on both things